internal/binutils/binutils.go

```diff
@@ -179,10 +179,20 @@ func (b *binrep) openMachO(name string, start, limit, offset uint64) (plugin.Obj
 	}
 	defer of.Close()
 
+	base := start
```

Comment:
I am a bit hesitant whether a better value for the base in the absence of __TEXT is zero or `start`. But I guess it's not important since files without __TEXT should be impractical case.

Reply:
Yeah I'll add a warning if it isn't found.